Dockerfile

```diff
@@ -134,6 +134,7 @@ RUN yum -y update postgresql-* && \
     yum clean all
 
 ADD container-assets/container-scripts /opt/manageiq/container-scripts/
+ADD container-assets/miq-run-postgresql /usr/bin/
```

Comment:
Super-super-minor, but I think we should move this into a separate "section", so it's not blended in with the upstream lines.  This way it's clear that it's ours and separate.

(i.e. move this down 3 lines into its own "paragraph" with a comment)